index.js

```diff
@@ -498,10 +498,18 @@ prototype.buildOperation = function(oldOperation, operationDefaults) {
     parameters.push(this.buildParameter(oldParameter));
   });
 
+  var tags = (operationDefaults.tags || []).concat(oldOperation.tags || []);
+
+  tags = tags.filter(function(e, i, arr) {
```

Comment:
Is this trying to find the last item in the array? Can't you just use `tags[tags.length - 1]` instead?